docs/building-apps/wallet/intro.mdx

```diff
@@ -60,6 +66,19 @@ val walletCustom = Wallet(
 )
 ```
 
+```typescript
+const customClient: AxiosInstance = axios.create({
+  baseURL: "https://some-url.com/api",
```

Comment:
Why do we even need to set baseUrl? I don't think it's good to show it in the example

Reply:
You don't necessarily, it's just an example of the params `axios.create` takes. I can rm